modules/social_features/social_profile/tests/src/Kernel/GraphQLUsersEndpointTest.php

```diff
@@ -99,7 +99,7 @@ static function (User $u) {
   public function testUserProfileFieldsPresence() : void {
     // Test as the admin users, this allows us to test all the fields that are
     // available in an all-access scenario.
-    $this->setCurrentUser(User::load(1));
+    $this->setUpCurrentUser([], [], TRUE);
```

Comment:
This achieves the same and no longer relies on user 1. The problems solved are the same as for our `Server::load` call.